learning_journal/routes.py

```diff
@@ -4,4 +4,6 @@ def includeme(config):
     config.add_route('detail', '/journal/{id:\d+}')
     config.add_route('update', '/journal/{id:\d+}/edit-entry')
     config.add_route('bootstrap', '/bootstrap/')
+    config.add_route('bootstrap_navbar', '/bootnav')
+    config.add_route('bootstrap_navbar2', '/bootnav/')
     config.add_static_view('static', 'static', cache_max_age=3600)
```

Comment:
You don't need to add a static view here if you're already doing it in your package's `__init__.py`